### ISSUE TITLE
Wolk: down

### ISSUE THREAD

**Issue by mitra42** [internetarchive/dweb-transports#19]
I'm seeing two issues with wolk.
a) URLs like  https://cloud.wolk.com/dweb.archive.org/metadata/netlabels are failing with "Key not found"
b) The library is returning that failure as a success, with a data structure that includes headers and a 404. 
I've disabled it for now in the default library.

**Comment by rodneywitcher** [comment]
Hi @mitra42 ... I've made changes (and sent a mail separately).  The issue lay in a code path that was only logging the error to the console, but not actually "throwing" it ... I've updated and tested on my end, but please do verify/confirm on your end.  Sorry about any trouble caused here.

**Comment by mitra42** [comment]
Thanks - I relabeled it, and its working at least some of the time (I'm trying to track down if failures are my end or yours) BUT - I am seeing a "DOM Exception" in loadAccounts, which I think is yours, can you confirm.

**Comment by rodneywitcher** [comment]
Yes, I'm seeing the "DOM Exception".  Investigating the cause, but as you said earlier, it doesn't seem to be causing any failures.  Something with the jwk export it seems.
